packages/loot-core/src/server/db/index.ts

```diff
@@ -580,22 +580,29 @@
   `);
 }
 
+const orphanedPayeesQuery = `
+  SELECT p.id
+  FROM payees p
+    LEFT JOIN payee_mapping pm ON pm.id = p.id
+    LEFT JOIN v_transactions_internal_alive t ON t.payee = pm.targetId
+  WHERE p.tombstone = 0
+    AND p.transfer_acct IS NULL
+    AND t.id IS NULL
+    AND NOT EXISTS (
+        SELECT 1
+        FROM rules r,
+        json_each(r.conditions) as cond
+        WHERE json_extract(cond.value, '$.field') = 'description'
+        AND json_extract(cond.value, '$.value') = pm.targetId
+    );
+`;
```

Comment:
table aliases are already in use with consistent syntax, and the subquery needs to stay as is because it must loop through the JSON in case the "description" field isn't the first index.